website/docs/reference/snapshot-configs.md

```diff
@@ -168,6 +168,24 @@ Configurations can be applied to snapshots using [YAML syntax](/docs/build/snaps
 
 <File name='dbt_project.yml'>
 
```

Comment:
grace mentioned event_time is supported for anythign that is ref-able. since snapshot is, i added event_time to it